src/data/markdown/docs/01 guides/02 Using k6/14 Executors.md

```diff
@@ -0,0 +1,295 @@
+| `maxDuration` | string  | Maximum test duration before it's forcibly stopped (excluding `gracefulStop`). | `"10m"` |
+
+
+### Constant VUs
+
+A fixed number of VUs execute as many iterations as possible for a specified amount
+of time. This executor is equivalent to the global `vus` and `duration` options.
+
+| Option     | Type    | Description                                     | Default |
+|------------|---------|-------------------------------------------------|---------|
+| `vus`      | integer | Number of VUs to run concurrently.              | `1`     |
+| `duration` | string  | Total test duration (excluding `gracefulStop`). | -       |
+
+
+### Ramping VUs
+
+A variable number of VUs execute as many iterations as possible for a specified
+amount of time. This executor is equivalent to the global `stages` option.
+
+| Option             | Type    | Description                                                                   | Default |
+|--------------------|---------|-------------------------------------------------------------------------------|---------|
+| `startVUs`         | integer | Number of VUs to run at test start.                                           | `1`     |
+| `stages`           | array   | Array of objects that specify the target number of VUs to ramp up or down to. | `[]`    |
+| `gracefulRampDown` | string  | Time to wait for iterations to finish before starting new VUs.                | `"30s"` |
+
+
+### Externally controlled
+
+Control and scale execution at runtime via k6's REST API or the CLI.
+
+This executor formalizes what was [previously possible globally](https://k6.io/blog/how-to-control-a-live-k6-test),
+and using it is required in order to use the `pause`, `resume`, and `scale` CLI commands.
+
+| Option     | Type    | Description                                         | Default |
+|------------|---------|-----------------------------------------------------|---------|
+| `vus`      | integer | Number of VUs to run concurrently.                  | -       |
+| `duration` | string  | Total test duration.                                | -       |
+| `maxVUs`   | integer | Maximum number of VUs to allow during the test run. | -       |
+
+
+### Per VU iterations
+
+Each VU executes a fixed number of iterations.
+
+| Option        | Type    | Description                                                                    | Default |
+|---------------|---------|--------------------------------------------------------------------------------|---------|
+| `vus`         | integer | Number of VUs to run concurrently.                                             | `1`     |
+| `iterations`  | integer | Number of script iterations to execute with each VU.                           | `1`     |
+| `maxDuration` | string  | Maximum test duration before it's forcibly stopped (excluding `gracefulStop`). | `"10m"` |
```

Comment:
This should be just after the `shared iterations` IMO

Reply:
:+1: Resolved in f99fbf3.